uniffi_bindgen/src/templates/macros.rs

```diff
@@ -66,12 +66,20 @@ use uniffi::UniffiMethodCall;
 {% match meth.throws() -%}
 {% when Some with (e) -%}
 {{ this_handle_map }}.method_call_with_result(err, {{ meth.first_argument().name() }}, |obj| -> Result<{% call return_type_func(meth) %}, {{e}}> {
+    {%- if meth.something_something_arc() -%}
     let _retval = {{ obj.name() }}::{%- call to_rs_call_with_prefix("obj", meth) -%}?;
+    {%- else -%}
+    let _retval = {{ obj.name() }}::{%- call to_rs_call_with_prefix("&*obj", meth) -%}?;
+    {%- endif -%}
```

Comment:
The `method_call_with_result` is doing the work of choosing between the two `handle_map` implementations, so this `else` block is being seen by both `Threadsafe` and non-`Threadsafe` objects.